app/components/point.jsx

```diff
@@ -35,15 +42,24 @@ const Point = forwardRef((props, ref) => {
       onMouseLeave={onMouseOut}
       ref={ref}
     >
-      <div className={classes.contentContainer}>
-        <div className={classes.informationGrid}>
-          {subject && <div className={cx(classes.sharedSubjectStyle, classes[vState + 'Subject'])}>{subject}</div>}
-          {description && (
-            <div className={cx(classes.sharedDescriptionStyle, classes[vState + 'Description'])}>{description}</div>
-          )}
-          {childrenWithProps}
+      {isLoading ? (
+        <>
+          <div className={`${classes.loadingAnimation} ${classes.loadingAnimationSubject}`} />
```

Comment:
Rather than creating separate div's, it is possible to apply these tryled to the subject and description divs? 

If not, can you make the first div one line tall, and the second div two lines tall - and only have 2 divs.  Make it look as much like an empty point as possible.

Reply:
I can consolidate them